app-i18n/translate-shell: version bump, fix dependency, remove old

ferki opened the pull request:
This PR bumps `app-i18n/translate-shell` to the latest version (0.9.6.9, released on 2018-12-23), and fixes [bug 680310](https://bugs.gentoo.org/680310) by simply replacing `app-editors/emacs` dependency with `virtual/emacs`.

I also decided to drop old releases in a follow-up commit.

Please review and merge this PR, or let me know how if anything needs to be changed.

monsieurp commented on app-i18n/translate-shell/translate-shell-0.9.6.9.ebuild:
```diff
@@ -0,0 +1,37 @@
+
+RDEPEND="
+	app-misc/rlwrap
+	>=sys-apps/gawk-4.0.2
```
This version is long gone. Please change this line to `sys-apps/gawk` instead.

ferki replied:
Thanks @monsieurp, I updated that dependency too, and squashed it into the bump commit.

mgorny commented on app-i18n/translate-shell/translate-shell-0.9.6.9.ebuild:
```diff
@@ -0,0 +1,37 @@
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+curl +bidi test tts"
```
Sorting ;-).

ferki replied:
Ha! It must have been a leftover of a `s/fribidi/bidi/` replacement. Fixed.

mgorny commented on app-i18n/translate-shell/translate-shell-0.9.6.9.ebuild:
```diff
@@ -0,0 +1,37 @@
+HOMEPAGE="https://www.soimort.org/translate-shell/"
+SRC_URI="https://github.com/soimort/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="public-domain"
```
https://github.com/soimort/translate-shell/blob/develop/LICENSE

GitHub says it's Unlicense. Please compare the text to be sure.

ferki replied:
Good catch, thanks! Website listed it as public domain, but the repo license and waiver files are indeed matching Unlicense text. Fixed.

ferki commented:
Meanwhile, translate-shell-0.9.6.10 has been also released, so I updated this PR with that latest version, and rebased it on top of current master branch, as well as addressing feedback on this PR.

Also, [bug 684048](https://bugs.gentoo.org/684048) has been reported, which seems like a test failure with `sys-apps/gawk-5*`, so I refined the `gawk` dependency to only allow `sys-apps/gawk-4*`, and then I'll check upstream about the issue.

mgorny commented on app-i18n/translate-shell/translate-shell-0.9.6.10.ebuild:
```diff
@@ -0,0 +1,37 @@
+		media-video/mplayer
+		)
+	)"
+DEPEND="${RDEPEND}
```
Do you really need all those deps to build a binpackage/install it? If not, please file a followup removing RDEPEND here.
